packages/demand-bloc-payment/src/providers/AdyenProvider.test.ts

```diff
@@ -308,25 +305,28 @@ describe('AdyenProvider', () => {
       expect(cardElement.handleAction).toHaveBeenCalledTimes(1)
     })
 
-    it('should do nothing if tokenizeHostedFields have not been called', () => {
-      provider.startThreeDSecureVerification()
-
-      expect(cardElement.handleAction).toHaveBeenCalledTimes(0)
+    it('should do nothing if tokenizeHostedFields have not been called', async () => {
+      try {
+        await provider.startThreeDSecureVerification()
+      } catch (e) {
+        expect(cardElement.handleAction).toHaveBeenCalledTimes(0)
+      }
```

Comment:
doesn't look correct compared to prev version
please use one of:
- https://jestjs.io/docs/expect#expectassertionsnumber
- https://jestjs.io/docs/expect#expecthasassertions

to ensure checks are executed for async code, especially in catch blocks

Reply:
This test was broken before on master and expect was never fulifield. In these two test cases there is a check performed on what happens when payment method is not defined meaning `provider.tokenizeHostedFields()` which sets it from response wasn't called. With given conditions what is returned from  `await provider.startThreeDSecureVerification()` is `Promise.reject(new AdyenError(errors[codes.AE04], codes.AE04))` so the current solution with try catch block is correct